Content.Server.Database/Model.cs

```diff
@@ -500,6 +501,20 @@ public sealed class ServerRoleUnban
         public DateTime UnbanTime { get; set; }
     }
 
+    [Table(nameof(PlayerId))]
+    public sealed class RoleTimer
```

Comment:
You should make a unique index over `(PlayerId, Role)` so that the DB can find these quicker. You'll want to add it in `OnModelCreating()`, see `Job` as an example:

```cs
            modelBuilder.Entity<Job>()
                .HasIndex(j => new { j.ProfileId, j.JobName })
                .IsUnique();
```